test/classes/phing/components/TargetTest.php

```diff
@@ -51,8 +51,8 @@ public function testHiddenTargets()
         exec($cmd, $out);
         $out = implode("\n", $out);
         $offset = strpos($out, 'Subtargets:');
```

Comment:
It seems to me that this line could be removed then.

Reply:
Gonna fix it and the other tests that broked.

Reply:
@siad007 How do I run tests locally?

I didn't found a `CONTRIBUING.MD`, as well. running `./.travis.sh` gave me:
```bash
carusogabriel@caruso:~/Code/phing$ sudo ./.travis.sh 
[sudo] password for carusogabriel: 
config-set succeeded

Auto-discover pear channels and upgrade ...
config-set succeeded
Updating channel "pear.php.net"
Channel "pear.php.net" is up to date
Channel "pear.phing.info" is already initialized
carusogabriel@caruso:~/Code/phing$
```

Reply:
@carusogabriel `.travis.sh` is only used/usable within the Travis CI context. https://github.com/phingofficial/phing/blob/master/README.md details how to run the tests.

Reply:
@siad007 This test started to fail due to `assertContains` doesn't work with the `$offset` parameter.